coalib/output/ConsoleInteraction.py

```diff
@@ -918,27 +929,58 @@ def print_bears(bears,
     :param show_params:      True if the parameters and their description
                              should be shown.
     :param console_printer:  Object to print messages on the console.
+    :param args:            Args passed to coala command.
```

Comment:
Something I usually forget to check :smile: IMO, this should be checked by some bear itself. Doesn't PEP8 styling take care of this of things?